src/lib/crypto/types.ts

```diff
@@ -1,10 +1,41 @@
 const secp256k1 = require('secp256k1')
 import { bech32 } from 'bech32'
-import { Epoch, Hash, HexString, Nanowits } from "../types"
+import { Balance, Epoch, Hash, HexString, Nanowits, UtxoMetadata } from "../types"
 
-import { fromHexString, isHexString, toHexString } from "../../bin/helpers"
+import { fromHexString, fromWits, isHexString, toHexString, whole_wits } from "../../bin/helpers"
 import { sha256 } from "./utils"
 
+export class Coins {
+    readonly pedros: Nanowits;
```

Comment:
As this is something that developers could need to read, I'd rename it as `nanowits` and I'll leave the easter egg "Pedro" only in `fromPedros`.